sdk/src/main/java/com/qonversion/android/sdk/Qonversion.kt

```diff
@@ -58,6 +58,14 @@ class Qonversion private constructor(
             return initialize(context, key, internalUserId, null, false, null)
         }
 
+        @JvmStatic
+        fun initialize(
+            context: Application,
+            key: String
```

Comment:
I think will be better to add optional string `key` and check for it instead using an empty string